data-access/tests/solrproxy_test.py

```diff
@@ -71,3 +71,8 @@ def get_data_series_list_test(self):
         result = self.proxy.get_data_series_list()
 
         print len(result)
+
+    def test_find_all_tiles_by_metadata(self):
+        result = self.proxy.find_all_tiles_by_metadata(['granule_s:19811114120000-NCEI-L4_GHRSST-SSTblend-AVHRR_OI-GLOB-v02.0-fv02.0.nc'], ds="AVHRR_OI_L4_GHRSST_NCEI")
```

Comment:
Does this file already exist in SCM?

Reply:
The file does not exist in SCM. Most of the test method in solrproxy_test.py assumes there is existing data that has been ingested in a running Solr instance. Not sure how to best handle this.